### ISSUE TITLE
sphere-mollweide example not working

### ISSUE THREAD

**Issue by fredj** [openlayers/openlayers#6722]
http://openlayers.org/en/master/examples/sphere-mollweide.html

**Comment by fredj** [comment]
```
a519d833165dfdacd2c77ce4c3abb843e7f36cd7 is the first bad commit
commit a519d833165dfdacd2c77ce4c3abb843e7f36cd7
Author: greenkeeper[bot] <greenkeeper[bot]@users.noreply.github.com>
Date:   Fri Mar 3 19:53:34 2017 +0000

    chore(package): update proj4 to version 2.4.3
    
    https://greenkeeper.io/
```

**Comment by ahocevar** [comment]
This is probably because proj4.js does not accept coordinates any more that exceed the world bounds when transformed to WGS84.

**Comment by ahocevar** [comment]
This is caused by proj4js/proj4js#256, which will be fixed in the next proj4.js release.

**Comment by fredj** [comment]
fixed with #7106
